File: development/sphinx-console-web/src/main/webapp/sphinx-console/app/model/SnippetConfigurationWrapper.js
Ext.define('sphinx-console.model.SnippetConfigurationWrapper', {
    extend: 'Ext.data.Model',
    idProperty: 'collectionName',
    fields: [
        {name: 'collectionName', type: 'string'}
    ],
    hasOne: [
        {model: 'sphinx-console.model.SnippetConfiguration', getterName: 'getSnippetConfiguration', setterName: 'setSnippetConfiguration', associationKey: 'snippetConfiguration', associatedName: 'snippetConfiguration' },
        {model: 'sphinx-console.model.CronScheduleWrapper', getterName: 'getCron', setterName: 'setCron', associationKey: 'cron', associatedName: 'cron' }
    ]
});

File: development/sphinx-console-coordinator/src/main/java/ru/skuptsov/sphinx/console/coordinator/model/SnippetConfigurationWrapper.java
package ru.skuptsov.sphinx.console.coordinator.model;

import java.io.Serializable;

public class SnippetConfigurationWrapper implements Serializable {
    private String collectionName;
    
    private SnippetConfiguration snippetConfiguration;
    
    private CronScheduleWrapper cron;

	public String getCollectionName() {
		return collectionName;
	}

	public void setCollectionName(String collectionName) {
		this.collectionName = collectionName;
	}

	public SnippetConfiguration getSnippetConfiguration() {
		return snippetConfiguration;
	}

	public void setSnippetConfiguration(SnippetConfiguration snippetConfiguration) {
		this.snippetConfiguration = snippetConfiguration;
	}

	public CronScheduleWrapper getCron() {
		return cron;
	}

	public void setCron(CronScheduleWrapper cron) {
		this.cron = cron;
	}
    
    
}
